### PR TITLE
Monorepo: More LRU updates and ESM Fixes

### PR DESCRIPTION
**Pull request by holgerd77**

### CODE REVIEW

File: packages/genesis/src/index.ts [hunk 1]
```diff
@@ -2,6 +2,12 @@ import { Chain } from '@ethereumjs/common'
 
 import type { GenesisState } from '@ethereumjs/util'
 
+import mainnetJSON from './genesisStates/mainnet.json'
```
Comment (g11tech) [inline packages/genesis/src/index.ts:5]: this is a bad idea because it loads all json files

**Review by g11tech — DISMISSED** [review]
lgtm

**Review by acolytec3 — APPROVED** [review]
Approved already :-)

**Comment by jochem-brouwer** [comment]
:tada:

**Comment by holgerd77** [comment]
We are getting there. :-)

**Pull request merged**